_pages/robot/2018.html

```diff
@@ -1,44 +1,50 @@
 ---
 layout: default
-title: 2017 Robot
-permalink: /robot/2017
+title: 2018 Robot
+permalink: /robot/2018
 stylesheets:
 - banners
-- steampunk
-scripts:
-- steampunk
 ---
 <main>
-    <h1>2017 Robot</h1>
-    <ul class="banners">
-        <li><img src="/assets/img/first.svg">2017<br> North VA Innovation in Control</li>
-        <li><img src="/assets/img/first.svg">2017<br> North MD Creativity Award</li>
-        <li><img src="/assets/img/first.svg">2017<br> Chesapeake Innovation in Control</li>
-    </ul>
-    <img src="/assets/img/robots/2017.jpg" alt="The 2017 robot being carried onto the ground.">
-    <p>The 2017 game is proving to be one of the most challenging to date for our experienced team. Designed by Disney, FIRST STEAMWORKS aims to challenge teams to create mechanisms capable of delivering 11 inch diameter gears to human players stationed on the Jules Verne inspired airship. Teams also have the task of shooting 5 inch diameter wiffle balls to add fuel to the boiler, sending steam to fuel the airship. For a more detailed description of the game, see the <a href="https://www.youtube.com/watch?v=EMiNmJW7enI">the official video reveal</a>.</p>
+    <h1>2018 Robot</h1>
+    <embed width="660" height="460"
+src="https://www.youtube.com/v/jqxHeneuZwA">
+    </embed>
```

Comment:
This is actually the first time I've seen the `<embed>` tag used. Given that we currently use `<iframe>`s, you should use that instead. Also, `<embed>`s have no closing tag.